app/services/api/v1/ecf/induction_records_query.rb

```diff
@@ -10,23 +10,43 @@ def initialize(updated_since: nil, email: nil)
     end
 
     def all
-      induction_records = InductionRecord
-        .current
-        .joins(:induction_programme, :preferred_identity, :participant_profile)
-        .merge(ParticipantProfile.ecf)
+      induction_records = relevant_induction_records
 
       if updated_since.present?
         induction_records = induction_records.where("induction_records.updated_at > ?", updated_since)
       end
 
       if email.present?
-        induction_records = induction_records.where(preferred_identity: { email: })
+        induction_records = induction_records.joins(:preferred_identity).where(preferred_identity: { email: })
       end
 
       induction_records
-        .group_by(&:preferred_identity_id)
-        .transform_values(&:first)
-        .values
+    end
+
+    def relevant_induction_records
```

Comment:
Could we include a comment about what this does and why it is different ie. not just `current` records any more?